packages/spacecat-shared-utils/src/url-helpers.js

```diff
@@ -85,7 +85,12 @@ function composeBaseURL(domain) {
 
 async function composeAuditURL(url) {
   const urlWithScheme = prependSchema(url);
-  const resp = await fetch(urlWithScheme);
+  const resp = await fetch(urlWithScheme, {
+    method: 'GET',
+    headers: {
+      'User-Agent': 'curl/7.88.1',
```

Comment:
adding curl to the user agent feels like we are announcing that we are a bot 😅 if this helps to circumvent any bot detection system, let's collect their names so that we can tell our customers not to use those so called "bot detection systems"

Reply:
I understand, but for now how can we run the audits ?